oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
 
 
-class InvalidXsrfTokenError(Exception):
+class InvalidXsrfTokenError(Error):
```

Comment:
Re: filing issues - it's up to you.

On Thu, Jun 30, 2016 at 12:25 PM Pat Ferate <notifications@github.com>
wrote:

> In oauth2client/contrib/appengine.py
> <https://github.com/google/oauth2client/pull/533#discussion_r69194640>:
>
> >
> >
> > -class InvalidXsrfTokenError(Exception):
> > +class InvalidXsrfTokenError(Error):
>
> Alright, makes sense. I'll go back and revise this PR to a generalized
> exception refactoring. Then file the few other issues that came up, and the
> associated PRs.
>
> If I do the PRs right away, should I still file the issues separately?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/google/oauth2client/pull/533/files/c21dd0d0fa9a0986cfe6001b399bd5be1b5bcc1c#r69194640>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPUc_ldHQLgDwsQSElj3kzlarKBu3buks5qRBgIgaJpZM4JBU6i>
> .
>